CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Grafana Mimir
 
+* [ENHANCEMENT] Helm: support metricRelabelings in the monitoring serviceMonitor resources
```

Comment:
```suggestion
* [ENHANCEMENT] Helm: support metricRelabelings in the monitoring serviceMonitor resources. #5340
```

Can you also move this line to the helm-specific changelog? [This is the file](https://github.com/grafana/mimir/blob/666a80d98a4fef00cf0685784bb6f9f7782619c2/operations/helm/charts/mimir-distributed/CHANGELOG.md#L1)

Reply:
I've moved the line to the helm-specific CHANGELOG and removed it from the root-level CHANGELOG. If the intention was for both to be kept, I can re-add.

Reply:
It's good as it is, thanks!